drivers/spi/spi_ll_stm32.c

```diff
@@ -643,6 +643,13 @@ static int transceive(const struct device *dev,
 
 	LL_SPI_Enable(spi);
 
+#if CONFIG_SOC_SERIES_STM32H7X
```

Comment:
Should it be `#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)` ?
(ie are we sure this is series specific and not related to the IP)

Reply:
I don't see anything similar in the errata of STM32U5 or H5 that also use this compatible. So I'd say this is series specific.